src/plugins/ucx/config.cpp

```
@@ -47,6 +67,21 @@ config::modifyAlways(std::string_view key, std::string_view value) const {
     }
 }
 
+void
+config::validateTlsCudaSupport() const {
```

Cppcheck (2.21.0)

[style] The function 'validateTlsCudaSupport' is never used.

(unusedFunction)